src/Sarif.Converters.UnitTests/TestData/FortifyFprConverter/ExpectedOutputs/OneResultBasic.sarif

```diff
@@ -123,6 +123,7 @@
         {
           "commandLine": "[REMOVED]insourceanalyzer.exe -scan -b The_Build_ID -machine-output -f C:\\projects\\myproject\\scans\\2018-01-01_12:00:00\\Scan.fpr -format fpr",
           "startTimeUtc": "2018-01-01T12:00:00.000Z",
+          "toolExecutionNotifications": [],
```

Comment:
>"toolExecutionNotifications": [], [](start = 10, length = 33)

This is a hack to make the TC pass. (for now). When i change the name of the property, it starts to get serialized into the json. I am unable to figure out why? (same hack in other two fortifyFpr tests).

Reply:
You need to rename the IsSerializeXXX properties in the Core\Invocation.cs

---
In reply to: [261023231](https://github.com/Microsoft/sarif-sdk/pull/1314#discussion_r261023231) [](ancestors = 261023231)